scripts/dockcross-manylinux-build-module-wheels.sh

```diff
@@ -61,12 +61,18 @@ chmod 777 $(pwd)/tools
 mkdir -p dist
 DOCKER_ARGS="-v $(pwd)/dist:/work/dist/ -v ${script_dir}/..:/ITKPythonPackage -v $(pwd)/tools:/tools"
 DOCKER_ARGS+=" -e MANYLINUX_VERSION"
+DOCKER_ARGS+=" -e LD_LIBRARY_PATH"
 # Mount any shared libraries
 if [[ -n ${LD_LIBRARY_PATH} ]]; then
   for libpath in ${LD_LIBRARY_PATH//:/ }; do
-	  DOCKER_ARGS+=" -v ${libpath}:/usr/lib64/$(basename -- ${libpath})"
+          DOCKER_LIBRARY_PATH="/usr/lib64/$(basename -- ${libpath})"
+	  DOCKER_ARGS+=" -v ${libpath}:${DOCKER_LIBRARY_PATH}"
```

Comment:
Should we use spaces consistently? One tab character sneaked in here.

Reply:
Sorry, should be fixed now.